contentScript.js

```diff
@@ -0,0 +1,6 @@
+var s = document.createElement('script');
+s.src = chrome.extension.getURL('courses.js');
+s.onload = function() {
+    this.remove();
+};
+(document.head || document.documentElement).appendChild(s);
```

Comment:
From what I read, it seems that there is some inconsistency with when some things are loaded. If you don't account for the correct timing (not the best way to put it), it will overwrite what your script does. I read that this ensures that it will always show up on the page correctly.

Then again, I could be wrong, I just trusted their word for it.